docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,180 @@
+# Deploy Nebula Graph with Docker Compose
+
+There are multiple ways to deploy Nebula Graph, but using Docker Compose is usually considered to be a fast starter.
+
+## Reading guide
+
+If you are reading this topic with the questions listed below, click them to jump to their answers.
+
+* [What do I need to do before deploying Nebula Graph?](#prerequisites)
+* [How to fast deploy Nebula Graph with Docker Compose?](#how-to-deploy)
+* [How to check the status and ports of the Nebula Graph service?](#check-the-nebula-graph-service-status-and-ports)
+* [How to check the data and logs of the Nebula Graph service?](#check-the-service-data-and-logs)
+* [How to stop the Nebula Graph Services?](#stop-the-nebula-graph-services)
+* [What are the other ways to install Nebula Graph?](#other-ways-to-install-nebula-graph)
+
+## Prerequisites
+
+* You have installed the following applications on your host.
+  
+  | Application    | Recommended version | Official installation reference                                    |
+  | -------------- | ------------------- | ------------------------------------------------------------------ |
+  | Docker         | Latest              | [Install Docker Engine](https://docs.docker.com/engine/install/)   |
+  | Docker Compose | Latest              | [Install Docker Compose](https://docs.docker.com/compose/install/) |
+  | Git            | Latest              | [Download Git for Linux](https://git-scm.com/download/linux)       |
+
+* If you are deploying Nebula Graph as a non-root user, grant the user with Docker-related privileges. For a detailed instruction, see [Docker docs: Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+
+* You have started the Docker service on your host.
+
+* If you have already deployed another version of Nebula Graph with Docker Compose on your host, to avoid compatibility issues，back up [the service data](#check-the-service-data-and-logs) if you need, and delete the `nebula-docker-compose/data` directory.
+
+## How to deploy
+
+1. Clone the `v2-preview` branch of the `nebula-docker-compose` repository to your host with Git.
+
+    ```bash
+    $ git clone --branch v2-preview https://github.com/vesoft-inc/nebula-docker-compose.git
```

Comment:
add a note or todo:

TODO: update the version `v2-preview` to a new release version. in case they use a unstable old version.

Reply:
Added a DON'T here to prevent users from applying the current version for production.